### PR TITLE
add notebook for review nb test

### PR DESCRIPTION
**Pull request by naliboff**

### CODE REVIEW

File: bin/review_nb_test.ipynb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (naliboff) [inline bin/review_nb_test.ipynb:1]: <p>This is working as intended</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/1/discussion/#comment-1532729749'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (ljhwang) [inline bin/review_nb_test.ipynb:1]: <p>This is a test.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/1/discussion/#comment-1532729933'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (mgouiza) [inline bin/review_nb_test.ipynb:1]: <p>This looks great</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/1/discussion/#comment-1532733703'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Comment by naliboff** [comment]
Closing this PR as it was just an initial test to see if Review NB worked.

**Pull request closed without merging**